src/components/single-result.js

```diff
@@ -0,0 +1,71 @@
+import { Sparklines, SparklinesLine, SparklinesReferenceLine } from 'react-sparklines';
+
+
+const SingleResult = ({result}) => {
+
+  console.log(result);
+
+  const city = result.name;
+
+  const temperatures = result.data.map((t) => {
+    return t.temperature;
+  });
+
+  const pressures = result.data.map((p) => {
+    return p.pressure;
+  })
+
+  const humidities = result.data.map((h) => {
+    return h.humidity;
+  })
+
+  const getAverage = (arr) => {
+    const sum = arr.reduce((acc, curr) => {
+      return acc + curr;
+    }, 0)
+    return Math.floor(sum / array.length);
+  }
+
+  
+  return (
+    <div className='results container'>
+      <div className='row'>
+        <div className='col'>
+          {city}
+        </div>
+      
+    
+        <div className='col'>
+          <Sparklines data={temperatures}>
+            <SparklinesLine color='#FFA500' />
+            <SparklinesReferenceLine type="avg" />
+           </Sparklines>
+            <p>{getAverage(temperatures)} F</p>
```

Comment:
You're effectively repeating this block three times - how could you make it more DRY?